[FIX] ./src/constants/errorMessages.ts ESLint 오류 해결

## Summary

github action중에서 `./src/constants/errorMessages.ts` 파일에서 ESlint 오류 난 문제를 해결합니다.

## Description
오류 메시지

```bash
Run npm run lint

> puangfilm@0.1.0 lint
> next lint .

Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry


./src/app/upload/_components/ImageUpload.tsx
2[7](https://github.com/GDSC-CAU/PuangFilm-FE/actions/runs/11682372412/job/32529356974#step:6:8):3  Warning: 'min' is assigned a value but never used. Allowed unused vars must match /^_/u.  @typescript-eslint/no-unused-vars

./src/constants/errorMessages.ts
13:1  Error: Delete `⏎`  prettier/prettier

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
Error: Process completed with exit code 1.
```

--- a/src/app/upload/_components/ImageUpload.tsx
+++ b/src/app/upload/_components/ImageUpload.tsx
@@ -24,6 +24,7 @@ interface ImageUploadProps {
 function ImageUpload({
   images,
   setImages,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   min = MIN_IMAGE_NUMBER,
   max = MAX_IMAGE_NUMBER,
 }: ImageUploadProps) {

--- a/src/constants/errorMessages.ts
+++ b/src/constants/errorMessages.ts
@@ -10,4 +10,3 @@ export const GENERATION_ERROR_MSG: string = '프로필을 생성하는 실패했
 export const GENERATION_ERROR_CHECK_MSG: string = '이미지를 다시 확인해주세요';
 export const UPLOAD_ERROR_MSG: string = '이미지 업로드에 실패했습니다.';
 export const UPLOAD_ERROR_CHECK_MSG: string = '이미지를 다시 확인해주세요';
-

